android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt

```diff
@@ -58,6 +61,7 @@ class BetterPlayerPlugin : FlutterPlugin, ActivityAware, MethodCallHandler {
     private var showPictureInPictureAutomatically: Boolean = false
     private val pipRemoteActions: ArrayList<RemoteAction> = ArrayList()
     private var isVideoPlaybackEnded: Boolean = false
+    private var mediaSessionConnector : MediaSessionConnector? = null
```

Comment:
Sorry.
I think, `mediaSessionConnector` is not used.
If you can, please remove unused variable.